student.rb

```diff
@@ -0,0 +1,13 @@
+# student.rb
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
+    super(id, age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] Your super method should not be making use of keyword arguments like you have used in the highlighted line, kindly see the code snippet below for corrections :+1: 
- code snippet
```ruby
super(age, name, parent_permission)
```